### ISSUE TITLE
Slick is interfering with contact form PHP validation

### ISSUE THREAD

**Issue by JakeEng** [kenwheeler/slick#2519]
> short description of the bug / issue, provide more detail below.

Contact form PHP validation fails to function correctly when combined with Slick's scripts. The contact form consists of two pages and Slick is used to turn the page. When Slick's scripts are removed, the validation functions correctly.

====================================================================

###### `[  paste your jsfiddle link here  ]`  

http://jsfiddle.net/qzcq1os3/

====================================================================

#### Steps to reproduce the problem

1. Create a HTML form  with PHP validation
2. Use Slick as a page turner for the form
3. Fill in form and push send

====================================================================

#### What is the expected behaviour?
After filling in the form and pushing the send button, an email should be sent to me. 

====================================================================

#### What is observed behaviour?
Validation fails and states that I need to enter my name.
When Slick is removed, validation works correctly and an email is sent.

====================================================================

#### More Details

- Which browsers/versions does it happen on? All 
- Which jQuery/Slick version are you using? Most recent
- Did this work before? No

**Comment by hailwood** [comment]
I've provided an answer to this issue on your question on Stack Overflow: http://stackoverflow.com/questions/39570892/why-does-my-js-slider-slick-interfere-with-php-validation-on-my-contact-form/39571890#answer-39571890

--
tldr: Forms aren't going to work with this library. Carousels aren't designed for forms.

**Comment by JakeEng** [comment]
Yeah, I saw. Thanks for looking into this for me, I'll mark your answer as accepted.

**Comment by leggomuhgreggo** [comment]
Cloning won't happen if you turn off infinite, which seems to me to be more in line with the nature of form completion. There are also other work arounds to customize how the validation/form submission is working either through preventDefault,  form field proxies, or adding/removing slides.

Hope that helps!

closing: support request
